cosmic-settings/src/pages/accessibility/magnifier.rs

```diff
@@ -69,9 +69,9 @@ impl Default for Page {
                     "{}%{}",
                     val,
                     if val == ZoomConfig::default().increment {
-                        " (Default)"
+                        format!(" ({})", fl!("default")).to_string()
```

Comment:
Since this is already within a formatter, this should be used instead https://doc.rust-lang.org/stable/std/macro.format_args.html